dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/utils/HttpTestUtils.java

```diff
@@ -12,12 +12,34 @@
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.HttpRequest;
 
+import java.net.http.HttpResponse;
+
 /**
  *
  */
 public class HttpTestUtils {
 
+    /**
+     * Call MCP server, and get the String response body
+     */
     public static String callMCP(LibertyServer server, String path, String jsonRequestBody) throws Exception {
         return new HttpRequest(server, path + "/mcp").requestProp("Accept", "application/json, text/event-stream").jsonBody(jsonRequestBody).method("POST").run(String.class);
-    };
-};
+    }
+
+    /**
+     * Call MCP server notification endpoint, and provide an expected response code. No response body is returned
+     * If the expected status code is different from the response code, and exception is thrown
+     */
+    public static void callMCPNotification(LibertyServer server,
+                                           String path,
+                                           String jsonRequestBody,
+                                           int expectedResponseCode) throws Exception {
+
+        new HttpRequest(server, path + "/mcp")
+                .requestProp("Accept", "application/json, text/event-stream")
+                .jsonBody(jsonRequestBody)
+                .method("POST")
+                .expectCode(expectedResponseCode)
+                .run(HttpResponse.class);
```

Comment:
`HttpRequest` does not support returning an `HttpResponse`. If everything is working, this won't cause an issue. If the call unexpectedly returns a body, you'll get a weird error that doesn't tell you what went wrong.

Ask it for a `String` instead and assert that it returns `null`.